src/aria/widgets/CfgBeans.js

```diff
@@ -1675,8 +1675,12 @@ module.exports = Aria.beanDefinitions({
                 },
                 "ariaLive" : {
                     $type : "json:Boolean",
-                    $description : "Specifies whether role=alert and aria-live=assertive are included (true) or omitted (false) on the container element (defaults to true, only used if waiAria is true)",
+                    $description : "Specifies whether the aria-live=assertive property is included (true) or omitted (false) on the container element (defaults to true, only used if waiAria is true)",
                     $default : true
+                },
+                "role": {
+                    $type : "json:String",
+                    $description : "Specifies the role to use on the container of the ErrorList, if any. It defaults to 'status' if ariaLive is true. It is only used if waiAria is true."
```

Comment:
Maybe you should specify what the defaut if ariaLive is false (no role).

Reply:
@fbasso Thank you for your comment. Yes, indeed, I have to mention the behaviour when `ariaLive` is false.